SpatialGDK/Source/SpatialGDK/Private/Tests/SpatialView/ViewDeltaTest.cpp

```diff
@@ -6,71 +6,3 @@
 
 #define VIEWDELTA_TEST(TestName) \
 	GDK_TEST(Core, ViewDelta, TestName)
-
-using namespace SpatialGDK; 
```

Comment:
I guess tests will be added in a later PR?

Reply:
Tests will be added later. I don't think these tests  were ever really useful though.